vit-servicing-station-lib/Cargo.toml

```diff
@@ -11,7 +11,7 @@ license = "MIT OR Apache-2.0"
 async-trait = "0.1.33"
 base64 = "0.12.1"
 time = { version = "0.3", features = ["parsing", "formatting"] }
-diesel = { version = "1.4.5", features = ["sqlite", "r2d2"] }
+diesel = { version = "1.4.5", features = ["sqlite", "r2d2", "64-column-tables"] }
```

Comment:
yeah, I wanted to avoid that... And started a refactor on the `Queryable` implementation on `FullProposalInfo` to avoid it, but it turned out to be a bit complicated and I threw it away. I'm not really super accustomed to diesel yet to be honest.